### DISCUSSION TITLE
Tiered data still shows in bucket

### DISCUSSION THREAD

**Discussion by swapkh91** [minio/minio#18470] [Q&A]
I created a tiering on my minio bucket, which moves data to a GCS bucket after 1 day.

I am able to see the tiered data on Google Cloud Storage bucket
![image](https://github.com/minio/minio/assets/5755405/917773b4-9c1e-4324-9699-178dd791be4a)

But I also see the data in my minio bucket and it shows the space still occupied by the objects
![image](https://github.com/minio/minio/assets/5755405/a7190020-5034-44bf-8493-19c4d51e3113)

![image](https://github.com/minio/minio/assets/5755405/d94455a7-13e9-42d3-904f-aa2b4c410ff7)

Isn't the minio bucket supposed to be empty if the data has been moved to GCS or am I missing something?

PS: I'm very new to this so my understanding might be wrong about tiering/hot-cold storage

**Comment by klauspost** [comment]
Tiering only moves the *data* and is transparent. MinIO will request tiered data upstream when clients request it.
